apps/ewallet_db/lib/ewallet_db/role.ex

```diff
@@ -25,15 +26,26 @@ defmodule EWalletDB.Role do
 
   defp changeset(%Role{} = key, attrs) do
     key
-    |> cast(attrs, [:name, :display_name])
-    |> validate_required(:name)
+    |> cast(attrs, [:priority, :name, :display_name])
+    |> validate_required([:name, :priority])
     |> unique_constraint(:name)
   end
 
   @doc """
   Creates a new role with the passed attributes.
   """
   def insert(attrs) do
+    last_role = Role |> Query.last(:inserted_at) |> Repo.one()
```

Comment:
Can we use `Ecto.Query.API.max(:priority)` instead? In case the last inserted happen to have a very high priority, so this one happens to get high priority by chance.